CONTRIBUTING.md

```diff
@@ -14,6 +14,15 @@ This project is a monorepo managed by [Lerna][lerna-home]. This means that each
 
 All commits you make should adhere to our commit guidelines. We use [conventional commits][conventional-commits] strategy with slight modification of our own - [@lmc-eu/commitlint-config][commitlint-config]. This is later used during release phase to determine how to bump the packages' version numbers based on commit history. 🚀
 
+Message of commit must contain specific phrases, like `<type>(<package>): <commit-message> #<issue-id>` eg.`Feat(web-react): React Modal implementation #DS-331`
+
+### Run before commit:
+
+1. Prettier `% yarn format:fix`
```

Comment:
There is a `yarn test` script in each package, which is grouping these scripts together into one. So you can run only one.

Also when you are in root, you can just run `yarn test` or `yarn packages:test` which will run `test` script in each package.

So I think that there should be something more general like "test your code before committing with `yarn test`". Or you should mention that we are using type linting, code listing, testing, and code formatting.